### DISCUSSION TITLE
[SOLVED] Import issue

### DISCUSSION THREAD

**Discussion by Canpy** [snipe/snipe-it#9310] [Q&A] [answered]
Hello,

i have uploaded a csv file and only 1 of 24 records get imported. I dont know why this happend, i have already tried everything to fix it but nothing has worked.

Solution:

I forgot a Asset Tag column in the .csv, the Asset Tag column is a required field.

**Comment by snipe** [comment, marked as answer]
Glad you got it sorted.
